### ISSUE TITLE
Concepts name incorrectly contain gender

### ISSUE THREAD

**Issue by jheeffer** [semio/ddf--unpop--wpp_pop#3]
https://github.com/open-numbers/ddf--unpop--world_population_prospects/commit/97608e59a7f6e9bbefa9721589c2234053885aad#diff-29c950e46477941fb6e5504b45b48b21

In the latest autogenerated commit, these concepts names say "Female" while the concept is used for any gender (male, female and both sexes). Is this because WPP doesn't contain a gender-agnostic concept name you can use, so you just pick one from the male/female/both sexes variable names?

**Comment by semio** [comment]
Right, WPP doesn't contain a gender-agnostic concept name. The names come from table names in source files, I extracted them to source/metadata.xls and etl script will use this file. I updated the file in latest commit and changed the column order, I think this is why concept names changed. 

Because in master these concept names are not gender-agnostic too (they are for both sexes), I am thinking about to manually add a new column in metadata.xls for correct concept names..
